src/mcts/search.cc

```diff
@@ -436,10 +490,13 @@ std::vector<std::string> Search::GetVerboseStats(Node* node) const {
         up = -up;
         std::swap(lo, up);
       }
-      *oss << (lo == up                                                ? "(T) "
-               : lo == GameResult::DRAW && up == GameResult::WHITE_WON ? "(W) "
-               : lo == GameResult::BLACK_WON && up == GameResult::DRAW ? "(L) "
-                                                                       : "");
+      *oss << (lo == up
```

Comment:
Likely chain of `if .. *oss << "(T)"; else if .. else if .. else` will look nicer.

Reply:
Makes sense; do you want me to update this within the scope of this PR, or either of us to create a separate PR for it?